playground/panda.config.ts

```diff
@@ -39,12 +39,27 @@ export default defineConfig({
                 _dark: '#FFFFFF4D',
               },
             },
+            complementary: {
+              value: {
+                base: '{colors.black}',
+                _dark: '{colors.white}',
+              },
+            },
           },
         },
       },
       recipes,
     },
   },
+  staticCss: {
+    css: [
+      {
+        properties: {
+          cursor: ['ew-resize', 'ns-resize', 'nwse-resize', 'nesw-resize'],
```

Comment:
not sure this is needed since they are used in the `playground/src/components/Preview/responsive-border.ts` file

Reply:
@astahmer It's for the `playground/src/hooks/useResponsiveView.ts` file.
When we're resizing, we change the body cursor. Show it shows that we're resizing.

![CleanShot 2023-07-24 at 09 45 54@2x](https://github.com/chakra-ui/panda/assets/30869823/33ee3be5-f1d5-44f5-a7fa-5813e939c677)

Reply:
![Screenshot 2023-07-24 at 10 49 30](https://github.com/chakra-ui/panda/assets/47224540/eaa70dbc-f60b-4d1a-883a-1d048e62095d)
yes but they are used here so panda should be able to extract them already, no ?

Reply:
You're right @astahmer